Service/ClientFinderInterface.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Trikoder\Bundle\OAuth2Bundle\Service;
+
+use Trikoder\Bundle\OAuth2Bundle\Model\Client;
+
+interface ClientFinderInterface
+{
+    /** @api */
```

Comment:
@toniperic Use the full docblock format when describing structural elements. For example:

```php
/**
 * @api
 */
```